conda-store-server/conda_store_server/server/views/api.py

```diff
@@ -109,18 +135,14 @@ def paginated_api_response(
     }
 
 
-@router_api.get(
-    "/",
-    response_model=schema.APIGetStatus,
-)
+@router_api_v1.get("/", response_model=schema.APIGetStatus)  # fmt: skip
+@router_api_v2.get("/", response_model=schema.APIGetStatus)  # fmt: skip
```

Comment:
Here and elsewhere: since I needed to introduce the v2 API (and potentially other versions later), I decided to use the existing mechanism for defining versioned routes, as opposed to using a third-party library or something like that, because I don't want to make too many changes.

However, it's quite verbose, so I make sure routes are oneliners. This makes it easier to see if there are any changes between API versions, and takes less space line-wise. Because these oneliners can be long, I add a comment for the formatter, so they are not changed later.

Reply:
Note: I'm not 100% sold on the oneliners idea because it requires changing existing routes, which might potentially introduce bugs if I'm not careful and requires reviewing those routes. However, we do have some endpoints, like `api_get_build_lockfile`, where the route declarations would take 40 LOC otherwise, so this does help.

Reply:
I haven't looked into whether it's possible, but it might be also cool to configure the formatter to skip formatting lines starting with `@router`, to avoid having to insert skips everywhere, which is error-prone and verbose.